python/tests/integration_tests/tpc-h.py

```diff
@@ -0,0 +1,863 @@
+from util import run_pipeline
```

Comment:
this is the third time we add this source code to the repository; we already have a full copy in benchmarks and in the sql compiler. In the compiler it's in a resource file tpch.sql.